### PR TITLE
Prerelease changes

### PR DESCRIPTION
**Pull request by tsujan**

### CODE REVIEW

**Comment by tsujan** [comment]
@yan12125 
I wanted to check it with you. I hope the versioning is OK, after recent changes.

**Comment by yan12125** [comment]
That's good, thanks!

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,14 @@
+qtermwidget-1.1.0 / 2022-04-15
+===============================
+ * Handled the zero history size.
+ * Removed an unused variable.
+ * Corrected some code indentations.
+ * Do not test PyQt on CI.
```
Comment (yan12125) [inline CHANGELOG:6]: PyQt is later re-enabled on CI after it's fixed. Maybe just remove this item?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+ * Added API documentation and followed naming convention.
+ * Return focus to terminal on closing search bar.
+ * moved to SIP 5.x wth PyQt.
```
Comment (yan12125) [inline CHANGELOG:9]: Capital "Moved ..." is better. All other items start with a capital word.

**Review by yan12125 — APPROVED** [review]
Tested and the version for PyQt bindings is correct. Here are some small issues in release notes.

File: CHANGELOG (hunk 2 above)
Comment (tsujan) [reply CHANGELOG]: Corrected. Thanks.

Comment (yan12125) [reply CHANGELOG]: Thanks!

**Pull request merged**